docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1245 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Complete dApp Tutorial
+
+Build a complete on-chain voting dApp
+
+---
+
+## Overview
+
+In this tutorial you will build a complete dApp, smart contracts and frontend that allows users to submit a vote. You will also deploy your own gas station that will be used to pay for the gas needed to interact with your dApp.
```

Comment:
In this tutorial we will be building a voting application on the Kadena blockchain. The application will consist of three distinct layers:

1. Smart contract (_back-end_)
2. Web app (_front-end_)
3. [Gas Station](https://medium.com/kadena-io/users-shouldnt-pay-for-gas-4df989ec8236) (_account owned by the voting application funding gas payments on behalf of the voters_)

Reply:
Is that Medium article the best link we have? It's from 2019.

Reply:
How about this one: https://medium.com/kadena-io/the-first-crypto-gas-station-is-now-on-kadenas-blockchain-6dc43b4b3836